qa/tasks/qemu.py

```diff
@@ -338,6 +338,18 @@ def _teardown_nfs_mount(remote, client, service_name):
             'sudo', 'systemctl', 'start', service_name
         ])
 
+def _setup_parent_cache(remote, client):
```

Comment:
Nit: let the `immutable_object_cache` task be responsible

Reply:
done